var/spack/repos/builtin/packages/libffi/package.py

```diff
@@ -32,6 +32,11 @@ class Libffi(AutotoolsPackage):
     patch("clang-powerpc-3.2.1.patch", when="@3.2.1%clang platform=linux")
     # ref.: https://github.com/libffi/libffi/pull/561
     patch("powerpc-3.3.patch", when="@3.3")
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/libffi/libffi/pull/764.patch?full_index=1",
```

Comment:
Is there a reason to prefer this URL over https://github.com/libffi/libffi/pull/764.patch?full_index=1? I ask out of ignorance. I don't know if one or the other is considered more canonical, I just know that there are more uses of `github.com` over `patch-diff.githubusercontent.com` in the spack repo (but that doesn't mean it's the correct choice necessarily).